kfinance/constants.py

```diff
@@ -76,6 +76,15 @@ class LatestPeriods(TypedDict):
     now: CurrentPeriod
 
 
+class IndustryClassification(StrEnum):
+    sic = "SIC"
+    naics = "NAICS"
+    nace = "NACE"
+    anzsic = "ANZSIC"
+    spcapiqetf = "SPCAPIQETF"
+    spratings = "SPRATINGS"
```

Comment:
```suggestion
class IndustryClassification(StrEnum):
    sic = "SIC"
    naics = "NAICS"
    nace = "NACE"
    anzsic = "ANZSIC"
    spcapiqetf = "SPCAPIQETF"
    spratings = "SPRATINGS"
    gics = "GICS"
```

So we will need to be careful with permissions later on but to me it does make sense to consolidate GICS here as that can then simplify things